test/e2e/storage/testpatterns/testpattern.go

```diff
@@ -112,10 +112,8 @@ var (
 	}
 	// DefaultFsDynamicPV is TestPattern for "Dynamic PV (default fs)"
 	DefaultFsDynamicPV = TestPattern{
-		Name:                   "Dynamic PV (default fs)",
-		VolType:                DynamicPV,
-		SnapshotType:           DynamicCreatedSnapshot,
-		SnapshotDeletionPolicy: DeleteSnapshot,
+		Name:    "Dynamic PV (default fs)",
+		VolType: DynamicPV,
```

Comment:
just want to verify after removing snapshot option here, what will be changed related t what tests will be triggered?

Reply:
I think nothing will changed since all other tests dont test snapshot.